s3-parallel-put

```diff
@@ -264,8 +265,10 @@ def putter(put, put_queue, stat_queue, options):
 
                     content_type = None
                     if options.content_type:
-                        if options.content_type == "guess":
+                        if options.content_type in ["guess", "magic"]:
```

Comment:
Logic looks good. For clarity, I'd suggest handling `guess` and `magic` as completely separate branches, something like:

```python
if options.content_type == "magic":
  # do magic stuff
elif options.content_type == "guess":
  # do guess stuff
else:
  content_type = options.content_type
```